test.py

```diff
@@ -0,0 +1,4 @@
+import subprocess

```

Comment:
Please add a brief comment at the top of this file explaining what it's for and consider renaming the file something like `git_test.py`